### PR TITLE
Added shared signup forms

### PR DESCRIPTION
**Pull request by sara-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   fixes issue with item
-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/_signup-default.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    margin-top: 3rem;
+`
+
+const LoginLink = styled.a`
+    color: var(--color-red);
+    text-decoration: none;
+    cursor: pointer;
+`
+
+const SignupDefault = props => {
```
Comment (oskar-binary) [inline src/components/custom/_signup-default.js]: Please destructure the props here 
```
const SignupDefault = ({ email_error_msg, email, handleInput ... }) => {
```

File: src/components/custom/_signup-default.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                        <LoginLink key={1} onClick={props.parameters.handleLogin} />,
+                    ]}
+                />
+            </NoteText>
+        </>
+    )
+}
+
+SignupDefault.propTypes = {
+    parameters: PropTypes.object,
```
Comment (oskar-binary) [inline src/components/custom/_signup-default.js]: and then you can add the destructured props here

File: src/components/custom/_signup-flat.js [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+    @media ${device.tabletL} {
+        display: none;
+    }
+`
+const handleExternalLink = () => {
+    window.open(deriv_app_url, '_blank')
+}
+
+const SignupFlat = props => {
```
Comment (oskar-binary) [inline src/components/custom/_signup-flat.js]: and destructure the props here too

File: src/components/custom/_signup-simple.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    @media ${device.mobileL} {
+        text-align: center;
+    }
+`
+
+const EmailButton = styled(Button)`
+    margin-left: 0.8rem;
+    height: 4rem;
+`
+const SignupSimple = props => {
```
Comment (oskar-binary) [inline src/components/custom/_signup-simple.js]: and here

**Comment by arash-fs** [comment]
Please check about the text alignment with Sheryl, Also ask for the background Image.

**Pull request merged**